src/@next/Tabs/Tabs.tsx

```diff
@@ -73,7 +90,13 @@ export const Tabs = React.forwardRef<HTMLDivElement, TabsProps>(function Tabs(
 
   return (
     <StyledTabsContainer ref={ref}>
-      <StyledTabHeaderContainer>
+      <StyledTabHeaderContainer
+        ref={tabsHeaderRef}
+        onScroll={handleScroll}
+        data-scroll-left={canScrollLeft}
+        data-scroll-right={canScrollRight}
+        data-scroll-both={canScrollLeft && canScrollRight}
```

Comment:
I think we can combine `data-scroll-left` and `data-scroll-right` : https://www.w3.org/TR/CSS2/selector.html#attribute-selectors